nomos-services/network/Cargo.toml

```diff
@@ -12,26 +12,29 @@ chrono = { version = "0.4", optional = true }
 overwatch-rs = { git = "https://github.com/logos-co/Overwatch", branch = "main" }
 multiaddr = "0.15"
 serde = "1.0"
+serde_json = "1.0"
 sscanf = { version = "0.4", optional = true }
 sled = { version = "0.34", optional = true }
 tokio = { version = "1", features = ["sync"] }
 tokio-stream = "0.1"
+tokio-util = "0.7.8"
 thiserror = "1.0"
 tracing = "0.1"
-rand = { version = "0.8", optional = true }
+rand = { version = "0.7.3", optional = true }
```

Comment:
Degraded `rand` because the `nym-sphinx` crate requires the lower version of `rand`.